js/send-mail.js

```diff
@@ -151,14 +151,16 @@ $(function () {
 
 	// cc/bcc toggling
 	$(document).on('click', '#new-message-cc-bcc-toggle', function() {
-		$('#new-message-cc-bcc').slideToggle();
-		$('#new-message-cc-bcc #cc').focus();
+		var newMessageCcBcc = $('#new-message-cc-bcc');
+		newMessageCcBcc.slideToggle();
+		newMessageCcBcc.find('#cc').focus();
 		$('#new-message-cc-bcc-toggle').fadeOut();
 	});
 
 	$(document).on('click', '#reply-message-cc-bcc-toggle', function() {
-		$('#reply-message-cc-bcc').slideToggle();
-		$('#reply-message-cc-bcc #cc').focus();
+		var replyMessageCcBcc = $('#reply-message-cc-bcc');
+		replyMessageCcBcc.slideToggle();
+		replyMessageCcBcc.find('#cc').focus();
```

Comment:
Possible, i will check it in a other pr. Reverted it here, as it has nothing todo with jscs. THX